approaching-a-problem/activity-breaking-down-a-problem.md

```diff
@@ -0,0 +1,97 @@
+# Activity: Breaking Down a Problem
+
+## Goal
+
+Our goal is to practice taking a problem, and pulling it apart into separate steps. When we practice this, our brains should consider how to describe a process, and chunk it into approachable steps. It should be challenging to slow down and think through these!
+
+## Activity Instructions
+
+Follow the instructions about how and when to fill in the prompts below. For each prompt, describe the process in smaller steps. Put your steps in a list format.
```

Comment:
```suggestion
Follow the instructions about how and when to fill in the prompts below. For each prompt, describe the process in smaller, detailed steps. Put your steps in a list format. Imagine a robot or a kid using these steps to solve each problem.
```

If these are meant to be very 'light' exercises, then the sentence I added isn't needed. However, if it's meant to have students really think about detailed steps in processes then the last sentence can be a fun/annoying brain teaser ("What do you mean by putting bread on a plate? Where did the bread come from? A whole loaf of bread? Add jelly to the bread? How do I do that? Just drop it on the bread with my hands!?!")

Reply:
I love it!

I'm going to change your suggestion and drop "robot," hope you don't mind! Otherwise I love it